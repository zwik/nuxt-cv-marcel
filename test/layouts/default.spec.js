import { shallowMount } from '@vue/test-utils'
import defaultLayout from '../../layouts/default.vue'

describe('default.vue', () => {
  it('should have a default template', () => {
    const wrapper = shallowMount(defaultLayout, {
      stubs: {
        Nuxt: true
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
