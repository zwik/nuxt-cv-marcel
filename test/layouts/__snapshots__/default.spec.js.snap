// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default.vue should have a default template 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
